### DISCUSSION TITLE
Select API: Clefs

### DISCUSSION THREAD

**Discussion by andre2007** [rism-digital/verovio#3845] [Ideas]
Hi,

the toolkit method `Select` allows to select a measure range. It would be really great, if I could also specify here the clef.

Use Case:
If have a MusicXml with G and F clef. I want to create a midi file which only contains the G clef.
Therefore I call the select API as following:

```
toolkit.select(`{"clefs":"1"}`);
toolkit.redoLayout();
```

If the MusicXml has for example 3 clefs and I want to render clef 1 and clef 3, maybe I could specify them with a comma
like this:

```
toolkit.select(`{"clefs":"1,3"}`);
toolkit.redoLayout();
```

Kind regards
André

**Comment by ahankinson** [comment]
A clef is a symbol that has no sound. Do you mean you want to select a staff?

**Comment by andre2007** [comment]
You are right. Exactly, I want to select a staff.
